exercises/list-ops/example.rb

```diff
@@ -0,0 +1,67 @@
+class ListOps
```

Comment:
Since this file is an example, I don't think using the "real" names is a problem. But no strong feelings about this, just my opinion. :)